web_external/models/UserModel.js

```diff
@@ -1,11 +1,12 @@
 isic.models.UserModel = girder.models.UserModel.extend({
     name: function () {
-        var name;
+        var name = this.get('name');
```

Comment:
Yes, I overlooked the fact that we use `isic.models.UserModel` even in a few places that aren't created by `User.filteredSummary`. I'll fix.